decidim-conferences/app/views/decidim/conferences/admin/conference_copies/_form.html.erb

```diff
@@ -1,10 +1,7 @@
 <%= append_javascript_pack_tag "decidim_conferences_admin" %>
 
 <div class="form__wrapper">
-  <div class="card" id="conferences">
-    <div class="item_show__header">
-      <h2 class="item_show__header-title"><%= title %></h2>
-    </div>
+  <div class="card pt-4" id="conferences">
     <div class="card-section">
       <div class="row column xlarge-8">
```

Comment:
I think the xlarge-8 class doesn't do anything here 
```suggestion
      <div class="row column">
```

Also in line 9 we have xlarge-4